packages/grafana-ui/src/components/Switch/Switch.tsx

```diff
@@ -27,10 +27,10 @@ export const Switch = React.forwardRef<HTMLInputElement, Props>(
       <div className={cx(styles.switch)}>
         <input
           type="checkbox"
-          disabled={disabled}
+          disabled={disabled || readOnly}
```

Comment:
As previous comment can't we continue to use `disabled` here?

Reply:
fixed